app/models/tasks/copy_image_task.rb

```diff
@@ -3,9 +3,37 @@ def start!
     self.options = task_options
     job = fixer_copy_file(options)
     self.job_id = job[:job][:id]
+
+    send_rexif_job
+
     save!
   end
 
+  def send_rexif_job
+    sns = Aws::SNS::Client.new
+    sns.publish({
+      topic_arn: ENV['REXIF_JOB_EXECUTION_SNS_TOPIC'],
```

Comment:
Is it worth guarding this entire `sns.publish()` call with a check for this ENV?